dash_app.py

```diff
@@ -178,17 +177,20 @@ def make_cell(item, skin_tone, gender, font_size):
         target = item
     if additional_emojis:
         return [
-            wrap_emoji(target, font_size),
-            dbc.Button('More',
-                       id={
-                           'type': 'more-button',
-                           'index': item['text']
-                       },
-                       className="me-1",
-                       size='sm',
-                       outline=True,
-                       color='dark',
-                       style={'margin-top': '1em'}),
+            html.Div(
+                children=[
+                    html.Div(wrap_emoji(target, font_size)),
+                    dbc.Button('More',
+                               id={
+                                   'type': 'more-button',
+                                   'index': item['text']
+                               },
+                               className="btn-secondary btn-sm")
+                ],
+                style={'display': 'flex', "gap": "20px", "align-items": "center",
+                       'justify-content': 'space-between', 'margin-right': '20pxx`'}
```

Comment:
Something (maybe justify?) is throwing the `more` button all the way to the right of the column, and the emoji and copy button all the way to the left. That looks ok when it's narrow on the web browser but I think I'd rather have the more button closer to the emojis with some padding or margin. Having it the right (or even left?) seems ok though.

Reply:
It's the `'justify-content': 'space-between'` CSS. I can change this to `'flex-start'` which will cause it to hug closer to the emojis